nyancat/config/genesis.json

```diff
@@ -0,0 +1,240 @@
+          }
+        ],
+        "critical_voting_period": "86400000000000",
+        "critical_max_num": "1",
+        "critical_threshold": "0.8340000000",
+        "critical_veto": "0.3340000000",
+        "critical_participation": "0.8572000000",
+        "critical_penalty": "0.0009000000",
+        "important_deposit_period": "86400000000000",
+        "important_min_deposit": [
+          {
+            "denom": "iris",
+            "amount": "100"
+          }
+        ],
+        "important_voting_period": "86400000000000",
+        "important_max_num": "2",
+        "important_threshold": "0.8000000000",
+        "important_veto": "0.3340000000",
+        "important_participation": "0.8340000000",
+        "important_penalty": "0.0007000000",
+        "normal_deposit_period": "86400000000000",
+        "normal_min_deposit": [
+          {
+            "denom": "iris",
+            "amount": "50"
+          }
+        ],
+        "normal_voting_period": "86400000000000",
+        "normal_max_num": "1",
+        "normal_threshold": "0.6670000000",
+        "normal_veto": "0.3340000000",
+        "normal_participation": "0.7500000000",
+        "normal_penalty": "0.0005000000",
+        "system_halt_period": "20000"
+      }
+    },
+    "upgrade": {
+      "GenesisVersion": {
+        "UpgradeInfo": {
+          "ProposalID": "0",
+          "Protocol": {
+            "version": "0",
+            "software": "https://github.com/irisnet/irishub/releases/tag/v0.14.1",
+            "height": "1",
+            "threshold": "0.9000000000"
+          }
+        },
+        "Success": true
+      }
+    },
+    "slashing": {
+      "params": {
+        "max_evidence_age": "86400000000000",
+        "signed_blocks_window": "2000",
+        "min_signed_per_window": "0.5000000000",
+        "double_sign_jail_duration": "600000000000",
+        "downtime_jail_duration": "600000000000",
+        "censorship_jail_duration": "600000000000",
+        "slash_fraction_double_sign": "0.0100000000",
+        "slash_fraction_downtime": "0.0000000000",
+        "slash_fraction_censorship": "0.0000000000"
+      },
+      "signing_infos": {},
+      "missed_blocks": {},
+      "slashing_periods": null
+    },
+    "service": {
+      "params": {
+        "max_request_timeout": "100",
+        "min_deposit_multiple": "1000",
+        "service_fee_tax": "0.0100000000",
+        "slash_fraction": "0.0010000000",
+        "complaint_retrospect": "1296000000000000",
+        "arbitration_time_limit": "432000000000000",
+        "tx_size_limit": "4000"
+      }
+    },
+    "guardian": {
+      "profilers": [
+        {
+          "description": "genesis",
+          "type": "Genesis",
+          "address": "faa108w3x8",
+          "added_by": "faa108w3x8"
+        }
+      ],
+      "trustees": [
+        {
+          "description": "genesis",
+          "type": "Genesis",
+          "address": "faa108w3x8",
+          "added_by": "faa108w3x8"
```

Comment:
Add some profilers and trustees?

Reply:
Without profilers, we cannot initiate SoftwareUpgrade proposals.

Reply:
Done